distsql/request_builder.go

```diff
@@ -183,7 +183,6 @@ func TableRangesToKVRanges(tid int64, ranges []*ranger.Range, fb *statistics.Que
 	if fb == nil || fb.Hist() == nil {
 		return tableRangesToKVRangesWithoutSplit(tid, ranges)
 	}
-	ranges = fb.Hist().SplitRange(ranges)
```

Comment:
Split it in `table_reader`?

Reply:
Removed by mistake when remove debugging log.

Reply:
Why should we move this `SplitRange` to `TableReaderExecutor::Open`?